### PR TITLE
Whitespace fix

### PR DESCRIPTION
**Pull request by dandelionred**
Just fixing some whitespace found with
```sh
find . -name \*.profile -exec grep -P '\s\s+' {} +
```

### CODE REVIEW

File: test/profiles/test.profile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-   blacklist /sbin/iptables
```
Comment (dandelionred) [reply test/profiles/test.profile]: Oops, looking into the file history I see it was intended for the sake of the parser testing.

I reverted changes to test/profiles/test.profile and test/profiles/test2.profile.

Comment (SkewedZeppelin) [inline test/profiles/test.profile]: these are probably intended for testing

File: test/environment/rlimit.profile [hunk 2]
```diff
@@ -1,5 +1,5 @@
-  rlimit-fsize 1024
```
Comment (SkewedZeppelin) [inline test/environment/rlimit.profile]: this one too

Comment (dandelionred) [reply test/environment/rlimit.profile]: Generic syntax checking for extra whitespaces is done here

https://github.com/netblue30/firejail/blob/f959fbb3a817dbf1452193b44efaebcc45e8cd83/test/profiles/profiles.sh#L18-L22

Hence this should only check for settings set via profile, not extra whitespace parsing

https://github.com/netblue30/firejail/blob/f959fbb3a817dbf1452193b44efaebcc45e8cd83/test/environment/environment.sh#L111-L112

Don't you think so?

**Comment by rusty-snake** [comment]
Any progress here?

**Comment by dandelionred** [comment]
> Any progress here?

I'm pretty sure test/environment/rlimit.profile should not include extra whitespace.

The only places to test for extra whitespace should be firejail/test/profiles/test.profile and firejail/test/profiles/test2.profile. So in case something goes wrong with extra whitespace parsing the point of failure should be syntax tests **only**, not rlimit test.

**Comment by rusty-snake** [comment]
@SkewedZeppelin ?

**Comment by SkewedZeppelin** [comment]
Merged, thanks.

**Pull request merged**